<launch>
  <node name="rplidarNode"          pkg="rplidar_ros"  type="rplidarNode" output="screen">
    <!--param name="serial_port"         type="string" value="COM3"/-->  
  <param name="serial_baudrate"     type="int"    value="256000"/><!--A3 -->
  <param name="frame_id"            type="string" value="laser"/>
  <param name="inverted"            type="bool"   value="false"/>
  <param name="angle_compensate"    type="bool"   value="true"/>
  <param name="scan_mode"           type="string" value="Sensitivity"/>
  <param name="topic"           	type="string" value="rplidarScan"/>
  <param name="useWIT"           	type="bool" value="true"/>
  <param name="useTilter"         type="bool" value="false"/>
  <param name="useRplidar"        	type="bool=" value="true"/>
    <!--by not including the serial ports we are saying the app should look them up automatically-->
    <!--include them only if you want to hard code the port (unlikely)-->
    <!--ports will be looked up automatically if the useService is true-->
    <!--param name="witdevice_serial_port"         type="string" value="COM9"/-->
  <!--param name="tilter_serial_port"         type="string" value="AUTO"/-->
  </node>
</launch>
